crowdloans/projects/kusama.json

```diff
@@ -210,7 +210,7 @@
     "website": "https://www.integritee.network/",
     "icon": "https://auction-us-east-2.s3.us-east-2.amazonaws.com/icons/chains/integritee.svg",
     "rewardRate": 10,
-    "hidden": false 
```

Comment:
we can delete it from all kusama projects

Reply:
Delete it the next time you update it.